core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.controller.ts

```diff
@@ -79,27 +79,28 @@ angular.module('oppia').directive('topicsAndSkillsDashboardPage', [
           var _initDashboard = function(stayInSameTab) {
             TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
               function(response) {
-                ctrl.topicSummaries = response.data.topic_summary_dicts;
+                ctrl.topicSummaries = response.topic_summary_dicts;
                 ctrl.editableTopicSummaries = ctrl.topicSummaries.filter(
                   function(summary) {
                     return summary.can_edit_topic === true;
                   }
                 );
                 ctrl.untriagedSkillSummaries =
```

Comment:
When you have multi-line statements, always wrap in `(...)` and break after `(` (see style guide [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)). Here and elsewhere in this file.